code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/style/AepButtonStyle.kt

```diff
@@ -37,6 +38,7 @@ class AepButtonStyle(
     var border: BorderStroke? = null,
     var colors: ButtonColors? = null,
     var contentPadding: PaddingValues? = null,
+    var buttonTextStyle: AepTextStyle? = null
```

Comment:
when updating the tests i saw that this is used like
```kotlin
buttonStyle.buttonTextStyle
```
what do you think of making the parameter name just `textStyle`?